index.html

```diff
@@ -9,6 +9,7 @@
 <body>
   <main>
     <p>This is a git-hub-flow test.</p>
+    <p>This is the second paragraph</p>
```

Comment:
You are submitting an older version of our index.html file. I'll try and add your line to the up to date file and see how it goes.